blueprints/node-webkit/index.js

```diff
@@ -50,5 +51,22 @@ module.exports = {
         return writeFile(packageJsonPath, JSON.stringify(json, null, '  '));
       }
     });
+  },
+  addQUnitLogger: function(){
+    var testHelperFile = path.resolve(this.project.root, 'tests/test-helper.js');
+    var _this = this;
+    
+    console.log('modifying tests/test-helper.js to require vendor/qunit-logger');
```

Comment:
Is there a logger we should use? Or a style of logging? I intentionally put this in here so that the user would know we're mucking with one of their code files.